src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -953,8 +957,8 @@ function updateNumScalePossibleValues(questionNumber) {
         $("#maxScaleBox"+idSuffix).val(max);
     }
     
-    if (step < 0.001) {
-        step = 0.001;
+    if (getFractionPart(step) < 0.001 && getFractionPart(step) != 0) {
```

Comment:
This looks like it's for restricting values with more than 3 decimal places, but does not seem like the right way of doing so. For example, if given "0.1001", this check will fail to detect that, because 0.1001 > 0.001, right? The original code also has this problem.